### ISSUE TITLE
Add a new page for useful services

### ISSUE THREAD

**Issue by matteomessmer** [matteomessmer/hacktoberfest#22]
It would be cool to have a page or to display generic services.
In this page there should be at least weather forecast and local news based on the location of the user.
The location could be approximated using the public ip.
I don't expect for just one person to do the whole page, so you can make just a little piece of it.
If you find other interesting stuff that could be added go for it!
Also if you find some cool APIs just comment it here.

**Comment by thelavenderhue** [comment]
@matteomessmer can I work on this? I'll make the Front-end

**Comment by amritanshusikdar** [comment]
@matteomessmer I can make the backend by fetching the weather data from [Open Weather Map](https://openweathermap.org/). Can I contribute?

**Comment by matteomessmer** [comment]
Yes sure

**Comment by matteomessmer** [comment]
I created services.html where to put this stuff, I think it should be better to just focus on one service for now and create other issues to add more. So yeah, just write here what you will do and then we see, or create a new issue and describe what you will work on.

**Comment by amritanshusikdar** [comment]
Yeah right, me and @thelavenderhue will work on just the Weather service. I guess it might be better to keep this particular issue open for ideas, and make subsequent new issues for each service. So it'd be better if a new issue is created where only the weather service shall be discussed.

**Comment by matteomessmer** [comment]
@amritanshusikdar nice idea, I'll open an issue for the weather and I'll assign you and @thelavenderhue to it.

**Comment by Sawsanalshaghel** [comment]
I'll try to work on the local news section

**Comment by antonio-b** [comment]
new to github, can I contribute?

**Comment by matteomessmer** [comment]
Yes sure

**Comment by Blaim1** [comment]
@matteomessmer I have forked the repo and tweaked the weather.css file for the "details-today" to look better.
Can you assign me this?

**Comment by amritanshusikdar** [comment]
@matteomessmer  I've added the API support for the weather service and also made a PR. Please check and let me know if its okay.
